pkg/manifests/manifests.go

```diff
@@ -22,3 +25,39 @@ func GetFileData(fileName string, output interface{}) error {
 
 	return err
 }
+
+type DecodeFormat interface {
+	NewDecodedYaml(decoder *yaml.YAMLToJSONDecoder) (interface{}, error)
+}
+
+// Read a YAML file containing multiple YAML definitions of the same format
+// Each specific format must be of type DecodeFormat
+func GetFileMultipleYamls(filename string, decoder DecodeFormat) ([]interface{}, error) {
+
+	path := filepath.Join("./manifests", filename)
```

Comment:
We'll need this dir to come from cobra flags soon. Please, put a TODO

Reply:
Added TODO in comment